doc/user/content/concepts/indexes.md

```diff
@@ -105,6 +105,100 @@ CREATE INDEX idx_on_my_view IN CLUSTER active_cluster ON my_view (...);
 
 ## Usage patterns
 
+### Index usage
+
+{{% important %}}
+Indexes are local to a cluster. Queries in a different cluster cannot use the indexes in another cluster.
+{{% /important %}}
+
+Unlike some other databases, Materialize can use an index to serve query
+results even if the query does not specify a `WHERE` condition on the index
+key.
+
+For example, consider the following index:
+
+```mzsql
+CREATE INDEX idx_orders_view_qty ON orders_view (quantity);
+```
+
```

Comment:
Here is where I'd put a sentence or two explaining indexes, e.g., "Materialize will maintain the `orders_view` in memory in `idx_orders_view_qty`, and it will be able to use the index to serve a variety of queries on the `orders_view`---and not just queries on `orders_view.quantity`."

Reply:
Added in patch.